cypress/integration/host/host_buttons.ts

```diff
@@ -12,8 +10,16 @@ describe("Host page restart jasper and update host status buttons", () => {
 
   it("Should show a toast when jasper restarted", () => {
     cy.dataCy("restart-jasper-button").click();
-    cy.get(popconfirmYesClassName).click();
+    cy.contains("button", "Yes").click();
+    cy.dataCy("toast").should("exist");
+    cy.get(`[aria-label="Close Message"]`).click();
+  });
+
+  it("Should show a toast when host is reprovisioned", () => {
+    cy.dataCy("reprovision-button").click();
+    cy.contains("button", "Yes").click();
     cy.dataCy("toast").should("exist");
+    cy.get(`[aria-label="Close Message"]`).click();
```

Comment:
Will these tests with toasts have to change given the refactor to toast component?